test/functional/test_framework/mininode.py

```diff
@@ -1310,8 +1310,8 @@ def __repr__(self):
 class msg_headers(object):
     command = b"headers"
 
-    def __init__(self):
-        self.headers = []
+    def __init__(self, headers=None):
+        self.headers = headers if headers is not None else []
```

Comment:
Just?
```py
    def __init__(self, headers=[]):
        self.headers = headers
```

Reply:
That is not safe in Python: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments.

Reply:
Oh! nice, ty!